node-tests/helpers/module-unification.js

```diff
@@ -0,0 +1,13 @@
+function enableModuleUnification() {
+  beforeEach(function() {
```

Comment:
Hi @Turbo87 

At #17212, I was enabling MU differently because that PR also upgraded `ember-cli-blueprint-test-helpers@0.19.1` which implements a new API described at ember-cli/ember-cli-blueprint-test-helpers#153

```
emberNew({ isModuleUnification: true })
```

1. Do we want to upgrade `ember-cli-blueprint-test-helpers` and use the new API? I can work on this PR but I would like to have a confirmation first.

2. In order the suggested PR to work, ember-cli/ember-cli#8240 needs to be merged.

Reply:
from what I can tell `emberNew({ isModuleUnification: true })` does basically the same thing as the `setupModuleUnification()` that I introduced here. feel free to update it. I don't know how https://github.com/ember-cli/ember-cli/pull/8240 is related to this though.